### DISCUSSION TITLE
Spline custom HTTP dispatcher not working

### DISCUSSION THREAD

**Discussion by rupesh3020** [AbsaOSS/spline-spark-agent#497] [Q&A] [answered]
Hi Team,

I tried creating a custom http dispatcher and using it while configuring the cluster.
I just added one security header property in splineHeaders.

I have attached the address of repository and cluster configurations and logs . 
https://github.com/rupesh3020/splinecustumhttpdispatcher

cluster conf :

spark.spline.lineageDispatcher http1
spark.spline.lineageDispatcher.http1.timeout.connection 1000
spark.spline.lineageDispatcher.http1.className org.abnamro.dmp.mdc.splinecustomhttpdispatcher.AzureHttpLineageDispatcher
spark.spline.mode REQUIRED
spark.databricks.delta.preview.enabled true
spark.spline.lineageDispatcher.http1.producer.url https://somefunc.azurewebsites.net/api/func/TEST
spark.spline.lineageDispatcher.http1.securityHeader.key functionkey

logs
[splineclusterlog.txt](https://github.com/AbsaOSS/spline-spark-agent/files/9639845/splineclusterlog.txt)

Please help me to solve this. thanks in advance :)

**Comment by cerveada** [comment, marked as answer]
Hello,

check the logs, you can see that there is a property that is not defined.

There are several default property values defined in `spline.default.properties`. If you copy the dispatcher you must defined this values for your copy as well.

**Reply by rupesh3020** [reply]
Solved the problem. Thanks :)
